converters/toZigbee.js

```diff
@@ -147,6 +147,35 @@ const converters = {
             await entity.read('genOnOff', ['onOff']);
         },
     },
+    TYZB01_on_off: {
+        key: ['state'],
+        convertSet: async (entity, key, value, meta) => {
+            if (typeof value !== 'object') {
```

Comment:
If I understand correctly now this only work via `{"state": {"state": "ON", "transition":1}}`, however `{"state":"ON","transition":1}` is preferred (and in line with others.. Change `key: ['state'],` to `key: ['state', 'transition'],` and access the state through `const state = meta.message.state` and `const transition = meta.message.transition`.  Also make sure this converter works with `{"state":"TOGGLE"}` ( `converters.on_off.convertSet(entity, key, value, meta);` should be called in this case)